src/client/components/ModalViewCancelEvent/ModalViewCancelEvent.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import Modal from '../Modal/Modal';
+import { Button } from '../Button/Button';
+
+export const ModalCancelEvent = () => {
+  const handleOnClick = () => {
+    return null;
+  };
```

Comment:
This doesn't do anything - you are supposed to pass `onClick` through `props` so that whoever is using this modal can do something after the button is clicked.